ui/app/templates/components/dashboard/secrets-engines-card.hbs

```diff
@@ -0,0 +1,69 @@
+<div class="is-flex-between has-left-margin-xs" data-test-dashboard-secrets-engines-header>
+  <h3 class="title is-4">Secrets Engines</h3>
+  <div>
+    <Hds::Link::Inline @route="vault.cluster.secrets.backends" class="has-text-weight-semibold is-no-underline">
+      View more
+    </Hds::Link::Inline>
+  </div>
+</div>
+<Hds::Table @caption="Five secrets engines" class="has-border-collapse" data-test-dashboard-secrets-engines-table>
+  <:body as |B|>
+    {{#each this.filteredSecretsEngines as |backend|}}
+      <B.Tr data-test-secrets-engines-row={{backend.id}}>
+        <B.Td class="is-flex-between is-flex-center">
+          <div>
+            <div class="is-flex-center">
+              {{#if backend.icon}}
+                <ToolTip @horizontalPosition="left" as |T|>
+                  <T.Trigger>
+                    <Icon @name={{backend.icon}} class={{unless backend.isSupportedBackend "has-text-grey"}} />
+                  </T.Trigger>
+                  <T.Content @defaultClass="tool-tip">
+                    <div class="box">
+                      {{or backend.engineType backend.path}}
+                    </div>
+                  </T.Content>
+                </ToolTip>
+              {{/if}}
+              {{#if backend.path}}
+                {{#if backend.isSupportedBackend}}
+                  <LinkTo
+                    @route={{backend.backendLink}}
+                    @model={{backend.id}}
+                    class="has-text-black has-text-weight-semibold is-no-underline"
+                    data-test-secret-path
+                  >
+                    {{backend.path}}
+                  </LinkTo>
+                {{else}}
+                  <span class="has-text-grey" data-test-secret-path>{{backend.path}}</span>
+                {{/if}}
+              {{/if}}
+            </div>
+            {{#if backend.accessor}}
+              <code class="has-text-grey is-size-8" data-test-accessor>
+                {{backend.accessor}}
+              </code>
+            {{/if}}
+            {{#if backend.description}}
```

Comment:
I'm not sure how we handle it on the normal secret engine list view, but try this out with a long description. It get's cutoff. You can add description under options when you first mount a secret engine. 
![image](https://github.com/hashicorp/vault/assets/6618863/e99bdb0c-f7f0-4669-82ec-65727be233bc)

![image](https://github.com/hashicorp/vault/assets/6618863/e88b1b1a-5dc6-492d-b6a8-becc1e0a869e)

Reply:
Ahh, I think Claire worked on this component. But we add a "See more" on the regular view. Probably too many links for this card, but maybe a `...` after a certain string length. Design might have some ideas. 
![image](https://github.com/hashicorp/vault/assets/6618863/0ec54a68-5326-4a39-a838-1ecb81beba1f)

Reply:
Good catch! I will ask Yongqi about this. For now, I will display the whole description and will ask Yongqi about how we want to tackle long descriptions.


<img width="530" alt="Screenshot 2023-06-20 at 1 46 38 PM" src="https://github.com/hashicorp/vault/assets/30884335/9deb5e03-9ffa-4932-b7ac-9a09320c9827">